src/AssetsPage/AssetsTable/index.jsx

```diff
@@ -226,24 +228,28 @@ export class AssetsTable extends React.Component {
   }
 
   render() {
-    return (!this.props.assetsList.length) ? (
-      <span>Loading....</span>
-    ) : (
-      <div>
-        {this.renderStatusAlert()}
-        <Table
-          columns={Object.keys(this.columns).map(columnKey => ({
-            ...this.columns[columnKey],
-            onSort: () => this.onSortClick(columnKey),
-          }))}
-          data={this.addSupplementalTableElements(this.props.assetsList)}
-          tableSortable
-          defaultSortedColumn="date_added"
-          defaultSortDirection="desc"
-        />
-        {this.renderModal()}
-      </div>
-    );
+    // TODO: Add UI for when there is nothing in the list and we have a status returned from the API
+    // TODO: http://fhtwd0.axshare.com/#g=1&p=files-and-uploads-empty
```

Comment:
These notes are for me to remember to check if we have a ticket set up for this case yet or not. Quick talk with Shelby makes it seem like this might not be necessary for feature parity, but good to keep track of for now.